lua/which-key/keys.lua

```diff
@@ -64,8 +64,7 @@ function M.process_motions(ret, mode, prefix, buf)
 
     if not ret.mapping and op_results.mapping then
       ret.mapping = op_results.mapping
-      ret.mapping.prefix = prefix
-      ret.keys = Util.parse_keys(ret.prefix)
+      ret.mapping.prefix = prefix_i  -- This is wrong but isn't used anywhere
```

Comment:
`ret` has type `MappingGroup`, of which `keys` is not a member. `ret.mapping.prefix` may be assigned a wrong value here, but it isn't used anywhere.